go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/NVIDIA/k8s-device-plugin
 
 go 1.21
+toolchain go1.22.0
```

Comment:
@ArangoGutierrez this is probably because of the versions in NFD. Since NFD is consumed by so many clients, does it make sense to drop the go version there slightly?